generated/nitclk/nitclk.proto

```diff
@@ -9,24 +9,17 @@ syntax = "proto3";
 option java_multiple_files = true;
 option java_package = "com.ni.grpc.tclk";
 option java_outer_classname = "NiTClk";
-option csharp_namespace = "NationalInstruments.Grpc.TClk";
```

Comment:
We can add this to hapigen's grpc_device/global_config/addition.hapigen like 'chsarp_namespace': NationalInstruments.Grpc.TClk' to get this generating like it used to.